Jormungand.DiscordReader-Mouth/src/main/java/entities/JorgMember.java

```diff
@@ -1,11 +1,17 @@
 package entities;
 
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Role;
 import net.dv8tion.jda.api.entities.User;
-import net.dv8tion.jda.internal.requests.Route;
 
 import java.util.List;
 
 public class JorgMember {
     User user;
-    List<String> roles; //Strings of JDA role names, role colours doesn't matter
+    List<Role> roles; //Strings of JDA role names, role colours doesn't matter
+
+    public JorgMember(Member member){
+        this.user = member.getUser();
+        this.roles = member.getRoles();
+    }
```

Comment:
JDA.Member contains a JDA api object so should break whenever `JorgMessageReceived` contains mentions